src/drawer.js

```diff
@@ -609,6 +610,9 @@ $.Drawer.prototype = {
     },
 
 
+    // private
+    _imageSmoothingEnabled: true,
```

Comment:
Here's a subtle JavaScript gotcha: by putting this property directly on the prototype like this, it means if any instance changes it, it gets changed for all instances. The fix is to make it part of the object directly by doing `this._imageSmoothingEnabled = true` in the constructor instead of having it here.

Reply:
OK, I'm realizing this gotcha doesn't apply to simple variables like booleans (it only applies to things like objects and arrays). That said, it would still be good to do the other way to keep everything together.